docs/cloud/data-ingestion.md

```diff
@@ -21,3 +21,101 @@ If your Graphite stack does not currently contain any relay, then you can simply
 
 When creating a Hosted Metrics Graphite instance, we provide a carbon-relay-ng config file that you can plug in and be ready to use out of the box.
 We also have Grafana Labs engineers ready to advise further on set up, if needed.
+
+
+## Sending carbon traffic to GrafanaCloud from an existing Graphite setup
+
+### Using carbon-relay-ng as a replacement for carbon-relay or carbon-cache
+
+The most simple way to send your carbon traffic to GrafanaCloud is to use carbon-relay-ng as a replacement for your current carbon-relay or carbon-cache. Carbon-relay-ng has a carbon input which supports the plain text and the pickle protocols, just like carbon-relay and carbon-cache. 
+
+An example configuration to do that can be downloaded from your Hosted Metrics instance's "Details" page. It contains a `grafanaNet` route pointing at your instance.
+
+### Sending a copy of your carbon traffic to GrafanaCloud
+
+It is possible to duplicate your data to send one copy to your existing Graphite infrastructure and the other to GrafanaCloud. To do this you can put an instance of carbon-relay-ng in front of your existing carbon-relay or carbon-cache and make it duplicate the traffic. Carbon-relay-ng allows you to specify routes of various types, to send a copy to GrafanaCloud you need to add a route of the type `grafanaNet`, to send a copy to your existing carbon-relay/carbon-cache you can add a carbon route.
+
+For example if you currently have carbon-relay listening on port `2003` and all of your infrastructure is sending its carbon traffic there, you could change it to listen on port `2053` instead and then start a carbon-relay-ng on port `2003` with this config to send a copy of the traffic to `localhost:2053`:
+
+```
+[[route]]
+key = 'carbon'
+type = 'sendAllMatch'
+destinations = [
+    'localhost:2053 spool=true pickle=false'
+]
+
+[[route]]
+key = 'grafanaNet'
+type = 'grafanaNet'
+addr = 'https://<Your endpoint address>'
+apikey = '<Your Grafana.com API Key>'
+schemasFile = '/etc/carbon-relay-ng/storage-schemas.conf'
+```
+
+### Duplicating the data in carbon-relay
+
+If you prefer to make your carbon-relay duplicate the carbon traffic and send one copy to carbon-relay-ng, this is possible if your current carbon-relay setup sends one copy of all the traffic to each of its known destinations. 
+
+For example if your carbon-relay currently forwards the traffic to two destinations called `host1` and `host2` and the replication factor (`relay.REPLICATION_FACTOR`) is set to the value `2`, then it is possible to just add carbon-relay-ng as an additional destination and increase the replication factor by `1` to create an additional copy of the traffic and send it to carbon-relay-ng:
```

Comment:
I don't think this is a common use. to shard across 2 hosts and have replication factor of 2
typically number of hosts will be much higher then replication factor. I think this is the scenario we should assume.
bumping RF just to get some traffic to relay-ng would affect the entire existing cluster. and also, relay-ng would only see a small portion of the traffic

Reply:
true, it's an unlikely scenario. I guess i should mention that this is not the recommended way of doing it. btw i just pushed again and now this section is much smaller between the others

Reply:
or do you think it's better to just remove this section? i guess maybe it's not necessary as we don't recommend it and it's unlikely to be applicable

Reply:
i removed it for now